frontend/src/modules/organization/helpers/organization.helpers.ts

```diff
@@ -34,22 +34,20 @@ const useOrganizationHelpers = () => {
   const emails = (organization: Organization) => (organization.emails || []);
 
   const primaryDomains = (organization: Organization) => organization.identities
-    .filter((i) => OrganizationIdentityType.PRIMARY_DOMAIN === i.type);
+    .filter((i) => OrganizationIdentityType.PRIMARY_DOMAIN === i.type && !['email'].includes(i.platform));
 
   const alternativeDomains = (organization: Organization) => organization.identities
-    .filter((i) => OrganizationIdentityType.ALTERNATIVE_DOMAIN === i.type);
+    .filter((i) => OrganizationIdentityType.ALTERNATIVE_DOMAIN === i.type && !['email'].includes(i.platform));
 
-  const domains = (organization: Organization) => organization.identities
-    .filter((i) => [
-      OrganizationIdentityType.PRIMARY_DOMAIN,
-      OrganizationIdentityType.ALTERNATIVE_DOMAIN,
-      OrganizationIdentityType.AFFILIATED_PROFILE,
-    ].includes(i.type));
+  const domains = (organization: Organization) => [
+    ...primaryDomains(organization),
+    ...alternativeDomains(organization),
+  ];
 
   const website = (organization: Organization) => primaryDomains(organization)?.[0];
 
   const affiliatedProfiles = (organization: Organization) => organization.identities
```

Comment:
If you are already removing affiliated profiles from the identities section in this PR, shouldn't you already show them in the attributes section?
If it's easier, we can also add an Affiliated profiles section below domains. We just should make sure that they are shown

Reply:
yes they are shown now, just had wrong type set in the helpers to get affiliated profiles